src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -71,6 +74,16 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
         return MONGO_CONNECTION_EXPECTED;
     }
 
+    public static validatePostgresConnectionString(value: string): string | undefined {
```

Comment:
Does the "pg-connection-string" package have anything that can do this for us?

Reply:
The pg-connection-string library only has parse function. I feel like it is unnecessary here for quick initial check.